### PR TITLE
refactor: notification service extension SERVICES-1107

### PR DESCRIPTION
**Pull request by johnxnguyen**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SERVICES-1107" title="SERVICES-1107" target="_blank">EpicSERVICES-1107</a>  Notification Service Extension
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

This is the feature branch for the notification service extension changes. It contains updates to account for the push token type when testing.

### Dependencies

Needs releases with:

- https://github.com/wireapp/wire-ios-transport/pull/243
- https://github.com/wireapp/wire-ios-utilities/pull/136

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: WireMockTransport.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -488,7 +488,9 @@
 				54C902341B7532DD007162A8 /* Products */,
 				54C902651B753419007162A8 /* Frameworks */,
 			);
+			indentWidth = 4;
```
Comment (agisilaos) [inline WireMockTransport.xcodeproj/project.pbxproj:491]: What that change is about? @johnxnguyen

**Review by agisilaos — APPROVED** [review]

File: WireMockTransport.xcodeproj/project.pbxproj (hunk 1 above)
Comment (KaterinaWire) [reply WireMockTransport.xcodeproj/project.pbxproj:491]: I don't think we need this changes. At least I don't remember why it was implemented. @johnxnguyen I can revert it. What do you think?

Comment (johnxnguyen) [reply WireMockTransport.xcodeproj/project.pbxproj:491]: It's just to set the project indentation setting to 4, as I have it set to 2 in my Xcode settings.

**Comment by johnxnguyen** [comment]
release major

**Comment by zenkins** [comment]
Hello, @johnxnguyen! Merging PR for command **release major** for PR #**223**. Will create a **major** release after merged.
Next version is 137.0.0.

**Pull request merged**